tests/system/small/test_pandas_options.py

```diff
@@ -289,7 +290,9 @@ def test_close_session_after_credentials_need_reauthentication(monkeypatch):
             bpd.read_gbq(test_query)
 
         # Now verify that closing the session works
-        bpd.close_session()
+        with warnings.catch_warnings(record=True) as warned:
```

Comment:
Please create a specific warning type for this in `bigframes.exceptions` and check for that here. See `UnknownLocationWarning`: https://github.com/search?q=repo%3Agoogleapis%2Fpython-bigquery-dataframes%20UnknownLocationWarning&type=code